livekit-agents/livekit/agents/utils/codecs/mp3.py

```diff
@@ -28,14 +31,12 @@ class Mp3StreamDecoder:
 
     def __init__(self):
         try:
-            globals()["av"] = import_module("av")
-        except ImportError:
+            self._codec = av.CodecContext.create("mp3", "r")  # noqa
+        except Exception:
```

Comment:
Can you leave it exactly the same as before?

```suggestion
try:
    import av
except ImportError:
  raise ImportError(
                "You haven't included the 'codecs' optional dependencies. Please install the 'codecs' extra by running `pip install livekit-agents[codecs]`"
            )
```

Reply:
that will make preloaded `av` unusued, I've updated code with 
```
        if importlib.util.find_spec("av") is None:
            raise ImportError(...)
```
what do you think?